### PR TITLE
Changes after validation

### PR DESCRIPTION
**Pull request by chloegugli**
Validation didn't succeed because:
-It was not possible to import pca workflows
-Groovy scripts were displayed as text in code mirror of catalog portal
-Some catalog objects didn't have extensions

### CODE REVIEW

File: app/scripts/proactive/config.js [hunk 1]
```diff
@@ -95,17 +95,17 @@ define(function () {
         },
         languages_content_type: {
             "scalaw" : "text/x-scala",
-            "groovy" : "application/x-groovy",
-            "javascript" : "application/javascript",
-            "python" : "application/x-python",
-            "cpython" : "application/x-python",
-            "ruby" : "application/x-ruby",
+            "groovy" : "text/x-groovy",
+            "javascript" : "text/javascript",
```
Comment (amouhoub) [inline app/scripts/proactive/config.js]: @chloegugli Why are they becoming text/xxxxx now? (I know it is a common debate on whether this is meant to be manipulated by machine or read by humans)

File: app/scripts/proactive/config.js [hunk 2]
```diff
@@ -95,17 +95,17 @@ define(function () {
...
+            "python" : "text/x-python",
+            "cpython" : "text/x-python",
+            "ruby" : "text/x-ruby",
             "r" : "text/x-R",
             "bash" : "application/x-sh",
```
Comment (amouhoub) [inline app/scripts/proactive/config.js]: depending on the logic upon which you changed the other from `application` to `text` MIME types, these ones should also follow that logic

File: app/scripts/proactive/config.js (hunk 1 above)
Comment (chloegugli) [reply app/scripts/proactive/config.js]: Actually this is a fix for "-Groovy scripts (and others) were displayed as text in code mirror of catalog portal". So I had a look at which content-types code mirror understands : https://codemirror.net/mode/

File: app/scripts/proactive/config.js (hunk 2 above)
Comment (chloegugli) [reply app/scripts/proactive/config.js]: After looking at which content-types code mirror understands (https://codemirror.net/mode/), some are 'text/' (groovy for example) and some are 'application' (powershell for example)

Comment (amouhoub) [reply app/scripts/proactive/config.js]: 👍

File: app/scripts/proactive/config.js (hunk 1 above)
Comment (amouhoub) [reply app/scripts/proactive/config.js]: 👍

**Pull request merged**